src/commands.json

```diff
@@ -0,0 +1,156 @@
+{
+	"TS.CREATE": {
+		"summary": "Create a new time-series",
+		"complexity": "O(1)",
+		"arguments": [{
+				"name": "key",
+				"type": "key"
+			},
+			{
+				"type": "block",
+				"name": "RETENTION",
+				"block": [{
+					"name": "retentionTime",
+					"type": "integer"
+				}],
+				"optional": true
+			},
+			{
+				"type": "block",
+				"name": "ENCODING",
+				"block": [{
+					"name": "ENCODING",
+					"type": "enum",
+					"enum": [
+						"UNCOMPRESSED",
+						"COMPRESSED"
+					]
+				}],
+				"optional": true
+			},
+			{
+				"type": "block",
+				"name": "CHUNK_SIZE",
+				"block": [{
+					"name": "size",
+					"type": "integer"
+				}],
+				"optional": true
+			},
+			{
+				"type": "block",
+				"name": "DUPLICATE_POLICY",
+				"block": [{
+					"name": "policy",
+					"type": "enum",
+					"enum": [
+						"BLOCK",
+						"FIRST",
+						"LAST",
+						"MIN",
+						"MAX",
+						"SUM"
+					]
+				}],
+				"optional": true
+			},
+			{
+				"type": "block",
+				"name": "LABELS",
+				"block": [{
+						"name": "label",
+						"type": "string"
+					},
+					{
+						"name": "value",
+						"type": "string",
+						"multiple": true
+					}
+				],
+				"optional": true
+			}
+		],
+		"since": "1.0.0",
+		"group": "timeseries"
+	},
```

Comment:
```suggestion
  "TS.CREATE": {
    "summary": "Create a new time-series",
    "complexity": "O(1)",
    "arguments": [
      {
        "name": "key",
        "type": "key"
      },
      {
        "type": "integer",
        "command": "RETENTION",
        "name": "retentionTime",
        "optional": true
      },
      {
        "type": "enum",
        "command": "ENCODING",
        "enum": [
          "UNCOMPRESSED",
          "COMPRESSED"
        ],
        "optional": true
      },
      {
        "type": "integer",
        "command": "CHUNK_SIZE",
        "name": "size",
        "optional": true
      },
      {
        "type": "enum",
        "command": "DUPLICATE_POLICY",
        "name": "policy",
        "enum": [
          "BLOCK",
          "FIRST",
          "LAST",
          "MIN",
          "MAX",
          "SUM"
        ],
        "optional": true
      },
      {
        "command": "LABELS",
        "name": [
          "label",
          "value"
        ],
        "type": [
          "string",
          "string"
        ],
        "multiple": true,
        "optional": true
      }
    ],
    "since": "1.0.0",
    "group": "timeseries"
  },
```